modules/effects/src/effects_module.ts

```diff
@@ -56,18 +70,49 @@ export class EffectsModule {
         EffectSources,
         Actions,
         rootEffects,
+        {
+          provide: _ROOT_EFFECTS,
+          multi: true,
```

Comment:
For root effects, we don't need `multi` as you can only call `forRoot` once.

Reply:
Alright, removed it